mm2src/coins/z_coin/z_rpc.rs

```diff
@@ -719,7 +671,9 @@ async fn light_wallet_db_sync_loop(mut sync_handle: SaplingSyncLoopHandle, mut c
         sync_handle.check_watch_for_tx_existence(client.as_mut()).await;
 
         if let Some(tx_id) = sync_handle.watch_for_tx {
-            if !block_in_place(|| is_tx_imported(sync_handle.wallet_db.db.lock().sql_conn(), tx_id)) {
+            let walletdb = &sync_handle.wallet_db;
+            let is_tx_imported = walletdb.is_tx_imported(tx_id).await;
```

Comment:
```suggestion
            let is_tx_imported = &sync_handle.wallet_db.is_tx_imported(tx_id).await;
```

Reply:
I thought it already seems understandable from the suggestion. Creating another variable doesn't seem to help in any way. The used field already points what its referring to.